docs/content/how-tos/rules/FL0076.md

```diff
@@ -0,0 +1,30 @@
+---
+title: FL0076
+category: how-to
+hide_menu: true
+---
+
+# FavourBasicControlFlow (FL0076)
+
+*Introduced in `0.21.3`*
+
+## Cause
+
+Detect the use of match clause when it is used in basic control flow.
```

Comment:
@janus remove the words "Detect the" (and make `use` become `Use` obviously).

Reply:
Done. Thanks.